e2e-tests/cypress/e2e/journeys/insurance/your-business/turnover/validation/turnover-estimated-annual-turnover-validation.spec.js

```diff
@@ -0,0 +1,159 @@
+import { turnover } from '../../../../../pages/your-business';
+import partials from '../../../../../partials';
+import { submitButton } from '../../../../../pages/shared';
+import { ERROR_MESSAGES } from '../../../../../../../content-strings';
+import { ROUTES, FIELD_IDS } from '../../../../../../../constants';
+import getReferenceNumber from '../../../../../helpers/get-reference-number';
+
+const TURNOVER_ERRORS = ERROR_MESSAGES.INSURANCE.EXPORTER_BUSINESS;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.prepareApplication.tasks.exporterBusiness;
+
+const {
+  TURNOVER: {
+    ESTIMATED_ANNUAL_TURNOVER,
+  },
+} = FIELD_IDS.INSURANCE.EXPORTER_BUSINESS;
+
+describe('Insurance - Your business - Turnover page - As an Exporter I want to enter details about my turnover - estimated annual turnover input validation', () => {
```

Comment:
What do you think about this format? I've been using the above for the "main page" E2E test but then for specific field tests i've been doing this

```js
context('Insurance - The section - The page - form validation - field name', () => {
```